### ISSUE TITLE
Deploy demo

### ISSUE THREAD

**Issue by dsvgl** [freddy38510/vue3-lazy-hydration#38]
The demo in this project is really nice. Would be great if it could be deployed to eg. Netlify or Vercel.
I think this would benefit the project and help people better understand hydration.

**Comment by freddy38510** [comment]
You are absolutely right. The demo is part of the project, it is handled by Vite.js in development mode. This means I can't build the distributable files to deploy to Netlify/Vercel.

I should instead create a separate project specifically to deploy the demo and provide a link to it in the documentation.
